README.md

```diff
@@ -4,46 +4,70 @@ Learn about [internship opportunities](https://www.ibm.com/quantum-computing/int
 
-### Quantum Engineer Internships
+_Coming soon: Quantum & Qiskit developer intern roles to be available at some of our international research labs including Zurich, Tokyo, and Haifa._
 
-Engineering interns will have the opportunity to focus on hardware or software engineering. Areas of engineering could include superconducting qubits, quantum software architecture, cryoelectronic circuit design, quantum microwave engineering, and quantum FPGA engineering (among other areas).
 
-We are not currently accepting new applications for IBM Quantum engineer internships.
+### Quantum Engineer Intern
 
+As an engineer with IBM Quantum, your job will be to advance the development of quantum computing systems. Areas of engineering include hardware, firmware, superconducting qubits, cryoelectronic circuit design, quantum microwave engineering, and quantum FPGA engineering (among other areas).
 
-### Quantum Community Advocate Internships
+Student engineers interested in working with IBM Quantum can apply to one of:
 
-Community advocate interns work on projects related to quantum education, community events, and open-source software, among other areas of focus. The ideal candidate must be able to work with internal and external stakeholders in order to deliver a positive experience for quantum computing community members.
+- [Quantum Engineer Summer Intern](https://careers.ibm.com/job/13791743/quantum-engineer-summer-intern-undergrad-2022-remote/) (Undergraduate)
+- [Quantum Engineer Summer Intern](https://careers.ibm.com/job/13791742/quantum-engineer-summer-intern-masters-phd-2022-remote/) (Master's/PhD)
 
-We are not currently accepting new applications for IBM Quantum community advocate internships.
+Quantum engineer summer interns will work at either the IBM Thomas J. Watson Research Center (Yorktown Heights, NY, USA) or one of the IBM Quantum labs in Rochester, MN, USA or Poughkeepsie, NY, USA for up to 12 weeks during the summer of 2022.
 
 
-### Quantum Designer Internships
+### Quantum Research Intern
 
-Design interns will have the opportunity to work on projects related to design research, user experience design, or visual design. They will work to uncover insights into the worldwide quantum computing community, and share quantum knowledge with users from the first introduction to quantum through quantum computing research and development.
+As an experimentalist, theorist, or quantum information scientist with IBM Quantum, your job will be to advance the field of quantum computing research. Areas of research include superconducting qubits, quantum control, quantum error correction, quantum complexity theory, quantum algorithms, quantum machine learning, and quantum compilers and optimizers (among other areas).
 
-We are not currently accepting new applications for IBM Quantum designer internships.
+Quantum Research Summer Intern (Master’s/PhD)
```

Comment:
Link missing for research intern job requisite.

Reply:
Updated!